optaplanner-quarkus-integration/optaplanner-quarkus/drools-integration-test/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-quarkus-parent</artifactId>
+    <version>8.6.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>optaplanner-quarkus-drools-integration-test</artifactId>
```

Comment:
Naming: It doesn't test CS-D, so "drools" is ambigious. It tests scoreDRL.

=> artifactId: optaplanner-quarkus-drl-integration-test
directory name: drl-integration-test